customer-commons/src/test/java/no/unit/nva/customer/model/ApplicationDomainTest.java

```diff
@@ -0,0 +1,26 @@
+package no.unit.nva.customer.model;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import java.net.URI;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+class ApplicationDomainTest {
```

Comment:
I guess this test is covered elsewhere now…try disabling it and see if it builds, if so, you can remove the test, migrating it to the place that actually covers it via the handler

Reply:
fixed